src/utils/aws-init/createRouteTable.js

```diff
@@ -0,0 +1,44 @@
+import {
+    EC2Client,
+    CreateRouteTableCommand,
+    CreateRouteCommand,
+    AssociateRouteTableCommand,
+} from '@aws-sdk/client-ec2';
+import addNametag from './addNametag.js';
+
+async function createRouteTable(VpcId, gatewayId, subnetIdArr) {
+    // Initialize the EC2 client
+    const client = new EC2Client({ region: 'us-east-1' });
+
+    // Create Route Table and specify parameters of new route table at the same time
+    const createRouteTableCommand = new CreateRouteTableCommand({
+        VpcId,
+        Main: true,
+    });
+
+    // Send request to initialize new route table
+    const createRouteTableResponse = await client.send(createRouteTableCommand);
+    const routeTableId = createRouteTableResponse.RouteTable.RouteTableId;
+
+    // adds each specified subnet to the route tables logs
+    subnetIdArr.forEach(async (id) => {
```

Comment:
Future optimization would be to make this run in parallel with `for await... of` instead of `forEach`